packages/MorphicAPIExplorer-Tests.package/MEXExplorationViewTests.class/instance/testCurrentMorphChanges.st

```diff
@@ -0,0 +1,8 @@
+tests
+testCurrentMorphChanges
+
+	| defaultMorphs view |
+	view := builder build: explorationView.
+	explorationView currentMorph: (ImageMorph new).
+	defaultMorphs := self getMorphCollectionInDefaultMorphBoxFor: view.
```

Comment:
I think you can use explorationView directly if we use [this variant](https://github.com/hpi-swa-teaching/MorphicAPIExplorer/blob/7dfd0e368d4f17d5bab5ea9cd3362f1ffca46371/packages/MorphicAPIExplorer-Tests.package/MEXExplorationViewTests.class/instance/getMorphCollectionInDefaultMorphBoxFor..st) as in #64

Reply:
the idea of the test is to check as well if the construction of the view is correct, thus I don't see any reason for bypassing the build process